package.json

```
@@ -22,10 +22,17 @@
   "exports": {
     ".": {
       "import": "./lib/index.mjs",
-      "require": "./lib/index.js"
+      "require": "./lib/index.js",
+      "types": "./types/index.d.ts"
```

```suggestion
      "types": "./types/index"
```
This would probably also need to be without the extension as mentioned in #14610

---

I need to do one last test but the current version should be correct. Whether the extension is present or not should follow `default` (or `require` / `import`)